README.md

```diff
@@ -2091,6 +2091,37 @@ _You can enable the following settings in Xcode by running [this script](resourc
 
     </details>
 
+* <a id='remove-blank-lines-between-chained-functions'></a>(<a href='#remove-blank-lines-between-chained-functions'>link</a>) **Remove blank lines between chained functions.** [![SwiftFormat: blanklinesbetweenchainedfunctions](https://img.shields.io/badge/SwiftFormat-blankLinesBetweenChainedFunctions-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/main/Rules.md#blanklinesbetweenchainedfunctions)
+
+  <details>
+
+  #### Why?
+
+  Improves readability and maintainability, making it easier to see the sequence of functions that are applied to the object.
+
+  ```swift
+  // WRONG
+  let planets = ["Mercury", "Venus", "Earth", "Mars"]
+
+  var transformedPlanetNames: [String] {
+      planets
+        .map { $0.uppercased() } // Making names uppercase
+        
+        .map { $0 + "!" } // Adding excitement with an exclamation mark
+  }
```

Comment:
Some thoughts on the example code:
 - Two sequential `map` calls is not common, since it would be simpler and more performant to just have a single map call like `map { $0.uppercased() + "!" }`. Instead, could we use `filter` and `map`?
 - I think we can omit the comments for simplicity (less is more!)
 - We can omit the `let planets = ["Mercury", "Venus", "Earth", "Mars"]` array since it isn't relevant to this specific rule.

```suggestion
  // WRONG
  var innerPlanetNames: [String] {
      planets
        .filter { $0.isInnerPlanet }
        
        .map { $0.name }
  }
```

We could _additionally_ have an example with code comments. That would help clarify the behavior of this [case](https://github.com/airbnb/swift/pull/272/files#r1631405303). I think it's good to address edge cases in separate examples, to keep the primary example nice and simple. For that example with comments, I think we should put the comments in between the method calls rather than on the right-hand side, since this formatting is more common imo.

Reply:
I hear what you're saying and agree. The example you list is more realistic, so I'll include that.